src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/DispatchBusinessProcessCallbackHandler.java

```diff
@@ -31,14 +34,30 @@
 
     @Override
     protected Map<String, Callback> callbacks() {
-        return Map.of(callbackKey(ABOUT_TO_SUBMIT), this::dispatchBusinessProcess);
+        return Map.of(
+            callbackKey(ABOUT_TO_START), this::checkIfBusinessProcessStarted,
+            callbackKey(ABOUT_TO_SUBMIT), this::dispatchBusinessProcess
+        );
     }
 
     @Override
     public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    private CallbackResponse checkIfBusinessProcessStarted(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        List<String> errors = new ArrayList<>();
+
+        if (caseData.getBusinessProcess().getStatusOrDefault() == STARTED) {
```

Comment:
Actually here I would be less strict with the check. What it Camunda already fully process the event, i.e. Status is then `FINISHED`... 

I would say better be more conservative, check that the status IS NOT `READY` instead?